core/data/src/main/java/uk/gov/gchq/gaffer/data/element/function/ExtractIdentifier.java

```diff
@@ -0,0 +1,88 @@
+/*
+ * Copyright 2017-2018 Crown Copyright
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package uk.gov.gchq.gaffer.data.element.function;
+
+import org.apache.commons.lang3.builder.EqualsBuilder;
+import org.apache.commons.lang3.builder.HashCodeBuilder;
+import org.apache.commons.lang3.builder.ToStringBuilder;
+
+import uk.gov.gchq.gaffer.data.element.Element;
+import uk.gov.gchq.gaffer.data.element.IdentifierType;
+import uk.gov.gchq.koryphe.Since;
+import uk.gov.gchq.koryphe.function.KorypheFunction;
+
+/**
+ * An {@code ExtractIdentifier} is a {@link KorypheFunction} for simplifying the process of extracting
+ * the object associated with a provided {@link IdentifierType}.
+ * If the Element is null, this will return null.
+ */
+@Since("1.4.0")
+public class ExtractIdentifier extends KorypheFunction<Element, Object> {
+    private IdentifierType id;
+
+    public ExtractIdentifier() {
+    }
+
+    public ExtractIdentifier(final IdentifierType id) {
+        this.id = id;
+    }
+
+    public IdentifierType getId() {
```

Comment:
Please can you put setters/getters at the bottom of the class. Similarly in your other class.